contrib/kafka/samples/README.md

```diff
@@ -8,6 +8,11 @@ sink.
 This sample demonstrates how to configure, deploy, and use the Apache Kafka
 Event Source with a Knative Service.
 
+For kubernetes, a simple Apache Kafka installation can be done with
```

Comment:
Should capitalize the `k` in Kubernetes.

Reply:
Fixed in f08bb45